### ISSUE TITLE
deal with `top.text = NULL` more cleverly

### ISSUE THREAD

**Issue by IndrajeetPatil** [IndrajeetPatil/statsExpressions#135]
How can this hideous code be refactored?!

https://github.com/IndrajeetPatil/statsExpressions/blob/b39471e06db8fe699247b8e116adaa386810ca66/R/add_expression_col.R#L140-L155

**Comment by IndrajeetPatil** [comment]
No longer relevant after #150.
